src/main/resources/twilio-node/operation.mustache

```diff
@@ -26,10 +26,54 @@
         const data: any = {};
 
         {{#queryParams}}
+        {{^vendorExtensions.x-serialize}}
         {{^required}}if (params.{{paramName}} !== undefined) {{/required}}data['{{{baseName}}}'] = params.{{paramName}};
+        {{/vendorExtensions.x-serialize}}
+        {{#isDate}}
+        {{^required}}if (params.{{paramName}} !== undefined) {{/required}}data['{{{baseName}}}'] = serialize.iso8601Date(params.{{paramName}});
```

Comment:
Can this logic be moved into the Java code? Basically, instead of setting various flags and vendor extension, just have the Java code set a single vendor extensions that says what the de/serialize function should be, if any. That would reduce a lot of the duplication and logic here.